src/lib/zf/tcp_out.c

```diff
@@ -956,7 +970,7 @@ int tcp_output(struct zf_tcp* tcp)
       else if( tcp_hdr->fin ) {
         flags |= ZF_PKT_REPORT_TCP_FIN;
       }
-      if( tcp_seg_seq(seg) != pcb->snd_nxt ) {
+      if( is_retransmit ) {
```

Comment:
This updated logic to check if a packet is a retransmission is purely so that users will see the "this is a retransmission" flag in the TX report. For context, please see the following excerpt from the passing log:
```
Sending Payload: 65
#### LONG SLEEP HAPPENS HERE ####
Count of timestamps:1
Timestamps: start:64, bytes:65, flags:1, seconds:1721381680 nsecs:401902045
Count of timestamps:1
#### NOTE flags & 0x2000 DENOTES RETRANSMIT ####
Timestamps: start:64, bytes:65, flags:2001, seconds:1721381700 nsecs:402331909
```